spyder/plugins/variableexplorer/widgets/preferences.py

```diff
@@ -0,0 +1,226 @@
+            )
+
+            self.by_column_button = self.create_radio_button(
+                _('Column by column'),
+                _(
+                    'Compare each cell against the largest and smallest '
+                    'numbers in the same column'
+                ),
+                comparator_group,
+                comparator_layout
+            )
+
+            main_layout.addWidget(comparator_group)
+
+        self.buttons = SpyderDialogButtonBox(
+            QDialogButtonBox.Ok | QDialogButtonBox.Cancel
+        )
+        main_layout.addWidget(self.buttons)
+        self.buttons.accepted.connect(self.accept)
+        self.buttons.rejected.connect(self.reject)
+
+        self.default_background_button.setChecked(True)
+
+        if type_string == 'dataframe':
+            self.global_button.setChecked(True)
+            comparator_group.setEnabled(False)
+            self.varying_background_button.toggled.connect(
+                lambda value: comparator_group.setEnabled(value)
+            )
+
+    @property
+    def float_format(self) -> str:
+        """
+        Format specification for floats.
+        """
+        return self.format_input.text()
+
+    @float_format.setter
+    def float_format(self, new_format: str) -> None:
+        self.format_input.setText(new_format)
+
+    @property
+    def varying_background(self) -> bool:
+        """
+        Whether to use a colored background.
+
+        If True, then vary the background of the cells in the editor according
+        to the value. If False, then use the default background in all cells.
+        """
+        return self.varying_background_button.isChecked()
+
+    @varying_background.setter
+    def varying_background(self, value: bool):
+        if value:
+            self.varying_background_button.setChecked(True)
+        else:
+            self.default_background_button.setChecked(True)
+
+    @property
+    def global_algo(self) -> bool:
+        """
+        Whether to use the global minimum and maximum to pick colors.
+
+        If True, then select the background color by comparing the cell value
+        against the minimum and maximum over the whole dataframe. If False,
+        then use the minimum and maximum over the column.
+
+        This attribute has no effect if `varying_background` is False.
+        """
+        if self.type_string == 'dataframe':
+            return self.global_button.isChecked()
+        else:
+            return True
+
+    @global_algo.setter
+    def global_algo(self, value: bool):
+        if self.type_string == 'dataframe':
+            if value:
+                self.global_button.setChecked(True)
+            else:
+                self.by_column_button.setChecked(True)
+
+    def add_help_info_label(self, layout, tip_text):
+        help_label = TipWidget(
+            tip_text=tip_text,
+            icon=ima.icon('question_tip'),
+            hover_icon=ima.icon('question_tip_hover')
```

Comment:
```suggestion
            hover_icon=ima.icon('question_tip_hover'),
            wrap_text=True,
```

I suggest to add this to not have very long tooltips when hovering/clicking on this widget, specially in other languages because most are not as terse as English.